gbus/builder/builder.go

```diff
@@ -171,7 +175,7 @@ func (builder *defaultBuilder) WithPolicies(policies ...gbus.MessagePolicy) gbus
 
 func (builder *defaultBuilder) Txnl(provider, connStr string) gbus.Builder {
 	builder.txnl = true
-	builder.txConnStr = connStr
+	builder.txConnStr = AddParseTimeToConnStr(connStr)
```

Comment:
@kerenbe4 we shouldn't change the conn string that is passed in by the user (no alarms an no surprises) as it effects the queries the user then does using the active transaction.
Please leave the connection string as is and deal with parsing the date time value out of the rows object in the saga store.
cc:@vladshub